generate-training-data.py

```diff
@@ -1,13 +1,27 @@
 import os
 import cv2
 
-# write your code here to open your laptop's camera
-# and store images for rock paper scissors
-
-# I would highly recommend storing these images in folders stucture like below:
-# training_data\
-#           |-- empty\
-#           |-- rock\
-#           |-- paper\
-#           |-- scissors\
-# having a folder structure like this will make it easy for you to read the images while pre-processing
+cap = cv2.VideoCapture(0)
+count = 0
+while (cap.isOpened()):
+    rep, frame = cap.read()
+
+    if rep == True:
+
+        #gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+        #out.write(frame)
+        font = cv2.FONT_HERSHEY_PLAIN
+        frame = cv2.rectangle(frame, (100,100), (400,400),(255,0,0), 5)
+        frame = cv2.putText(frame, "count="+str(count/5), (300, 420), font,1 , (255, 0, 0), 3)
+        if count%5 == 0:
+            cv2.imwrite('scissor' + str(count/5) + '.jpg', frame[100:400,100:400])
```

Comment:
Try to automate this task through code and also try following the folder structure I mentioned.

To automate this you can do this by running the same script multiple times by passing the name as a variable, either by running the file multiple times or by calling this script in a function and calling the function multiple times.

Try getting the user's input for that variable from via terminal, you can use `system arguments`, `argparser` or even the `input()` method to achieve this.

If this is not possible at least try storing the generated images into a folder and not in the root directory.

Reply:
Asking you to try doing this because the deadline for this task is till tomorrow 12:30 PM IST